components/supervisor-api/info.proto

```diff
@@ -101,8 +101,7 @@ message WorkspaceInfoResponse {
     // configcat_enabled controls whether configcat is enabled
     bool configcat_enabled = 18;
 
-    // default_workspace_image is the default image of the workspace
-    string default_workspace_image = 19;
+    reserved 19;
```

Comment:
Should we live a comment here, what was removed?
I also wonder about backward compatibility we cannot break with it anyone?